examples/sumo/pedestrian_traffic_light_grid.py

```diff
@@ -0,0 +1,252 @@
+"""Traffic Light Grid example."""
```

Comment:
Suspect this script could just be a set of command line flags in the regular traffic light grid env. Less maintenancet that way than adding another script.

Reply:
fixed